docs/src/content/docs/guides/customization.mdx

```diff
@@ -449,3 +449,28 @@ main {
   font-family: 'IBM Plex Serif', serif;
 }
 ```
+
+Finally, add your custom CSS file to Starlight’s `customCss` array in `astro.config.mjs` after the CSS file containing your `@font-face` declarations:
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight'
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs With a Custom Typeface',
+      customCss: [
+        // Path to your @font-face CSS file (if using local fonts)
+        '/src/fonts/font-face.css',
+        // Path to your @font-face CSS file (if using Fontsource)
+        '@fontsource/ibm-plex-serif/400.css',
+        '@fontsource/ibm-plex-serif/600.css',
+        // Path to your custom CSS file
+        '/src/styles/custom.css'
+      ],
+    }),
+  ],
+});
+```
```

Comment:
Thanks for this feedback! Do you think we can link to the existing guide to adding custom CSS here instead? The order of font-face and custom CSS files is unimportant, so the process shown there should be sufficient. Maybe something like this:

````suggestion
Follow the [custom CSS instructions](#custom-css-styles) to add your styles to your site.
````

Reply:
Hey! Sure thing, linking to the CSS section can definitely work if you prefer.

I realize the added markup is a bit verbose/redundant but I wasn't sure if it was left out on purpose (folks new to Astro might forget to add the stylesheet entirely).

But if linking is sufficient then that's okay.